src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
I had edited my comment to say I'll add a layer of deprecation.

The helpers add nothing on top of `<a-event>`. They can actually limit developers to think they can only handle a limited number of events, when actually they are able to handle any event. Plus, some of them are misleading because they don't map to the actual event name.